<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>GUI_History</class>
    <widget class="QDialog" name="GUI_History">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1214</width>
                <height>816</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QFrame" name="header">
                            <property name="frameShape">
                                <enum>QFrame::StyledPanel</enum>
                            </property>
                            <property name="frameShadow">
                                <enum>QFrame::Raised</enum>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="Gui::MenuToolButton" name="toolButton">
                            <property name="text">
                                <string notr="true"/>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QStackedWidget" name="stackedWidget">
                    <property name="currentIndex">
                        <number>1</number>
                    </property>
                    <widget class="QWidget" name="page">
                        <layout class="QGridLayout" name="gridLayout">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item row="1" column="0">
                                <widget class="QScrollArea" name="scrollArea">
                                    <property name="widgetResizable">
                                        <bool>true</bool>
                                    </property>
                                    <widget class="QWidget" name="scrollAreaWidgetContents">
                                        <property name="geometry">
                                            <rect>
                                                <x>0</x>
                                                <y>0</y>
                                                <width>98</width>
                                                <height>28</height>
                                            </rect>
                                        </property>
                                    </widget>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="page_2">
                        <layout class="QGridLayout" name="gridLayout_2">
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item row="2" column="0">
                                <widget class="QScrollArea" name="scrollAreaRange">
                                    <property name="widgetResizable">
                                        <bool>true</bool>
                                    </property>
                                    <widget class="QWidget" name="scrollAreaRangeContents">
                                        <property name="geometry">
                                            <rect>
                                                <x>0</x>
                                                <y>0</y>
                                                <width>1194</width>
                                                <height>726</height>
                                            </rect>
                                        </property>
                                    </widget>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QLabel" name="labFrom">
                                            <property name="text">
                                                <string notr="true">From</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_3">
                                            <property name="text">
                                                <string notr="true">-</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="labUntil">
                                            <property name="text">
                                                <string notr="true">Until</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer_2">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnClear">
                                            <property name="text">
                                                <string notr="true">Clear filter</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>Gui::MenuToolButton</class>
            <extends>QPushButton</extends>
            <header>Gui/Utils/MenuTool/MenuToolButton.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
